File: src/com/androhi/androiddrawableviewer/AndroidDrawableViewerToolWindowFactory.java
package com.androhi.androiddrawableviewer;

import com.intellij.openapi.project.Project;
import com.intellij.openapi.wm.ToolWindow;
import com.intellij.openapi.wm.ToolWindowFactory;
import com.intellij.ui.content.Content;
import com.intellij.ui.content.ContentManager;

public class AndroidDrawableViewerToolWindowFactory implements ToolWindowFactory {

    @Override
    public void createToolWindowContent(Project project, ToolWindow toolWindow) {
        DrawableViewer viewer = new DrawableViewer(project);
        final ContentManager contentManager = toolWindow.getContentManager();
        final Content content = contentManager.getFactory().createContent(viewer, null, false);
        contentManager.addContent(content);
    }
}

File: src/main/kotlin/com/androhi/androiddrawableviewer/form/AndroidDrawableViewerToolWindowFactory.kt
package com.androhi.androiddrawableviewer.form

import com.intellij.openapi.project.Project
import com.intellij.openapi.wm.ToolWindow
import com.intellij.openapi.wm.ToolWindowFactory

class AndroidDrawableViewerToolWindowFactory : ToolWindowFactory {

    override fun createToolWindowContent(project: Project, toolWindow: ToolWindow) {
        val drawableViewer = DrawableViewer(project)
        val contentManager = toolWindow.contentManager
        val content = contentManager.factory.createContent(drawableViewer, null, false)
        contentManager.addContent(content)
    }
}